packages/showcases/svelte/stories/components/indicators/VtmnBadge/docs.mdx

```diff
@@ -0,0 +1,51 @@
+import { Preview, Story } from '@storybook/addon-docs/blocks';
+
+# Badge
+
+## Installation and usage
+
+You need to add `@vtmn/svelte` on your `package.json`
+
+```
+npm install @vtmn/svelte
+```
+
+Now, you can use `VtmnBadge` on your svelte file with
+
+```svelte
+<script>
+  import { VtmnBadge } from '@vtmn/svelte';
+</script>
+```
+
+# Stories
+
+## Default
+
+<Preview>
+  <Story id="components-indicators-vtmnbadge--default" />
+</Preview>
+
+## Brand
+
+<Preview>
+  <Story id="components-indicators-vtmnbadge--brand" />
+</Preview>
+
+## Reversed
+
+<Preview parameters={{ backgrounds: { default: 'primary-reversed' } }}>
+  <Story id="components-indicators-vtmnbadge--reversed" />
+</Preview>
+
+## Accent
+
+<Preview>
+  <Story id="components-indicators-vtmnbadge--accent" />
+</Preview>
+
+## Alert
+
+<Preview>
+  <Story id="components-indicators-vtmnbadge--alert" />
+</Preview>
```

Comment:
Interesting :)

Reply:
We need to rollback storybook svelte to v6.3.13, Storybook have an issue with the generated template of svelte on ^6.4.
After the rollback, we can change the documentation to be more clear and commit the PR :)